src/languages/es.ts

```diff
@@ -336,6 +337,7 @@ const translations = {
             pleaseSelectOne: 'Seleccione una de las opciones.',
             invalidRateError: 'Por favor, introduce una tarifa válida.',
             lowRateError: 'La tarifa debe ser mayor que 0.',
+            email: 'Please enter a valid email address.',
```

Comment:
I'm waiting for final translations - https://expensify.slack.com/archives/C01GTK53T8Q/p1730378501391549